charts/veritable-cloudagent/templates/statefulset.yaml

```diff
@@ -332,8 +332,10 @@ spec:
           livenessProbe:
             exec:
               command:
-              - ipfs
-              - version
+                - '/bin/sh'
+                - '-c'
+                - |
+                  [ "$(ipfs swarm peers | wc -l)" -ge {{ if .Values.ipfs.initConfig.bootNodeAddress }}0{{ else }}2{{ end }} ]
```

Comment:
I think you are nearly there.
`.Values.ipfs.initConfig.bootNodeAddress` will only evaluate as true if an address is present, which would signify this is *not* a bootnode, in which case we want the value to be 2 and the `else` to be 0.

Reply:
Done